README.md

```diff
@@ -420,7 +420,7 @@ spec:
           name: okta-api-token
           namespace: group-sync-operator
         url: "https://example.okta.com/"
-        appId: okta-sync-app-id
+        appId: 0oad5lTSBOMUBOBVVQSC
```

Comment:
@michaelryanmcneill Can this be made generic just to confirm that it is not a real application ID?